core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1723,6 +1723,17 @@ public String toString() {
           .setIsHidden(true)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_SKIP_ROOT_ACL_CHECK =
+      new Builder(Name.MASTER_SKIP_ROOT_ACL_CHECK)
+          .setDefaultValue(false)
+          .setDescription("Skip root directory ACL check when restoring from backup. This is to "
```

Comment:
Is this only intended for backup restore? The code change affects any journal replay (restart, secondary catchup, backup restore).

Reply:
The only real use I see for this is in the event of restoring from a backup. However this can be helpful in other scenarios i.e. starting Alluxio as a different OS user if the old user had been deleted.

Also, I don't think this takes effect on secondary catchup since the check is done in isPrimary()

Reply:
I think this description is misleading, since this will actually affect all journal replay (restarts, secondaries, backup), not only backup. Can you update the description?